config/install/webspark_webdir.settings.yml

```diff
@@ -1,10 +1,10 @@
 api: ''
 departments: webdir-departments
-people_departments: webdir-people-in-department
-people_search: webdir-search-faculty-staff
-people_data: webdir-profiles-by-department
-people_department: webdir-people-in-department
-profile_affiliations: webdir-faculty-profile-affiliations
+people_departments: webdir-departments/webdir-profiles
+people_search: webdir-profiles/faculty-staff
+people_data: webdir-profiles/department
+people_department: webdir-departments/webdir-profiles
```

Comment:
Apologies. We need to change the new endpoint on lines 3 and 6 from `webdir-departments/webdir-profiles` to `webdir-departments/profiles` instead. This is the one that was missing from the ASUIS-388 pull request, and I decided to drop the second webdir- from the endpoint path so it would better match what we're doing with other resources (like the one on line 5).

Reply:
@mlsamuelson I changed the endpoint.